client/my-sites/subscribers/types/index.ts

```diff
@@ -24,5 +24,7 @@ export type Subscriber = {
 	email_address: string;
 	avatar: string;
 	display_name: string;
+	subscriptions: SubscriptionPlan[];
```

Comment:
I might have missed something, but why is `subscriptions: SubscriptionPlan[];` necessary here?